scripts/postgres/restoreDbFromSource.ts

```diff
@@ -215,8 +215,61 @@ async function restoreFromBackup({
   });
   const targetPassword = await targetSigner.getAuthToken();
 
+  // pg_restore only drops tables that exist in the source dump, so we drop all target tables before calling pg_restore.
+  // We could drop the whole target db or the schema, but then we would have to deal with stricter permissions.
   await new Promise(resolve => {
-    const result = spawn(
+    // For each table in the target db public schema, we will create a SQL DROP command and then execute it.
```

Comment:
Small piece of feedback: this new step in the script is lumped into the `restoreFromBackup` function. The `main` function may be more explicit about what is happening step by step if it called a new function named something to the effect of `dropAllTables`, instead of the dropping happening as a result of calling `restoreFromBackup`.

Reply:
Good call. I will do this as a follow-up :)

Reply:
(Want to make sure this gets delivered before the next glue job, and I've tested as is.)